docs/useStorage.md

```diff
@@ -1,46 +1,51 @@
 
    - Providers
     - Consumers
     #
 
-**Providers** is to define and keep track of specific pieces of state. This state can then be accessed by all the child components nested inside the Provider. These child components, known as Consumers, are responsible for accessing or modifying the state provided by the Context Provider.Points to notes about Context before choosing the right library.<br>
+**Providers** is to define and keep track of specific pieces of state. This state can then be accessed by all the child components nested inside the Provider. These child components, known as **Consumers**, are responsible for accessing or modifying the state provided by the Context Provider.Points to notes about Context before choosing the right library.<br>
 # 
 - Context is typically used for smaller-scale state management needs or for passing down data that is needed by many components in the application.
 - Context it's build into React itself, so there's no need to install additional libraries to use it.<br>
 
 
-And on the flip side we have **Redux** which provides us with a different way of viewing states in our application
+And on the flip side we have **Redux** which provides us with a different way of viewing states in our application.<br>
+
  
-Redux simplifies the way we view state in our application by requiring us to store all the state data in a single object. Everything we need to know about the application is in one place. with reduce we pull state management away from react entirely. 
+Redux simplifies the way we view state in our application by requiring us to store all the state data in a single object. Everything we need to know about the application is in one place. with redux we pull state management away from react entirely. 
 
 * The reduce API has three main concepts 
     - The store
     - The Reducer
     - Action
-    
-#
+
 The **store**: An immutable object that holds the application's state<br>
 **Reducer**: A function that returns some data, triggered by an action type<br>
 **Action**: An object that tells the reducer how to change the state it must contain a type property and optional payload property.<br>
 Points to note are.<br>
-#
-- Redux is typically used for lager-scale management needs or for applications with complex state logic that require features such as as middleware, time-travel debigging and predictable container state.
 
-To make the most use of the useStorage hook we are going to be using the Context library here this is because of its simplicity, also the Context api is a build in feature of react, the Context is suitable for managing global state or application-level state and also for the scope of our applications since we donot have to much data to handle.<br>
+- Redux is typically used for lager-scale management needs or for applications with complex state logic that require features such as as middleware, time-travel debigging and predictable container state.<br>
```

Comment:
Why are even talking about redux anyway?

Reply:
talking about Context and redux is to justify the choice of using Context instead of redux sir @stephane-segning